cmd/crio/config.go

```diff
@@ -263,8 +263,8 @@ image_volumes = "{{ .ImageVolumes }}"
 network_dir = "{{ .NetworkDir }}"
 
 # Paths to directories where CNI plugin binaries are located.
-plugin_dir = [
-{{ range $opt := .PluginDir }}{{ printf "\t%q,\n" $opt }}{{ end }}]
+plugin_dirs = [
+{{ range $opt := .PluginDirs }}{{ printf "\t%q,\n" $opt }}{{ end }}]
 `))
 
```

Comment:
We will have to retain the older plugin_dir as well, since that's what is used in the MCO template. See https://github.com/openshift/machine-config-operator/blob/6276485e84c53e6531adb540ce494e30170f5876/templates/worker/01-worker-container-runtime/_base/files/crio.yaml#L252

Reply:
I opted to append it to PluginDirs in lib/config validate(), so it does appear in the config because it had already been added to PluginDirs

Reply:
oh you mean print it. Ok